air-script/tests/aux_trace/aux_trace.masm

```diff
@@ -68,6 +95,16 @@ proc.compute_evaluate_integrity_constraints
     padw mem_loadw.4294900202 movdn.3 movdn.3 drop drop ext2mul
 end # END PROC compute_evaluate_integrity_constraints
 
+# Procedure to evaluate each integrity constraints.
```

Comment:
nit: should be "Procedure to evaluate numerators of all boundary constraints."?

Reply:
changed